INSTALL.md

```diff
@@ -176,10 +176,13 @@ eopkg install fontconfig-devel
 #### NixOS/Nixpkgs
 
 The following command can be used to get a shell with all development
-dependencies on [NixOS](https://nixos.org).
+dependencies using [Nix](https://nixos.org).
 
 ```sh
 nix-shell -A alacritty '<nixpkgs>'
+
+# or, if you're using Nix Flakes:
+nix develop nixpkgs#alacritty
```

Comment:
We should suggest one or another, Alacritty isn't an instruction manual on how to use your package manager, if you have troubles you're expected to look into this yourself.

I'm not sure what the status of the deprecation is, but if the `nix develop` option is already supported widely the existing one should just be replaced.

Reply:
Got it - in this case let's stick with current `nix-shell`; `nix develop` is still experimental and might change. I'll re-create this pull request in the future, when the command gets stabilized 🙂